OurUmbraco.Site/Views/MacroPartials/Project/ApiKeys.cshtml

```diff
@@ -0,0 +1,77 @@
+﻿@using OurUmbraco.Auth
+@inherits Umbraco.Web.Macros.PartialViewMacroPage
+@{
+    
+    var projectId = Request["Id"];
+    int id = 0;
+}
+
+@if (!string.IsNullOrEmpty(projectId) && int.TryParse(projectId, out id))
+{
+    var project = Umbraco.TypedContent(id);
+    var member = Members.GetCurrentMember();
+
+    if (project.GetPropertyValue<int>("owner") != member.Id)
```

Comment:
Can only owners create keys, collaborators are not allowed at this time?

Reply:
Yes currently that is how it works, may extend to collabs later on 🙂